bin/init

```diff
@@ -12,6 +12,7 @@ fi
 
 mkdir -p /data/grafana
 chown -R grafana /data/grafana
+chown -R grafana /usr/share/grafana
```

Comment:
Grafana 2.1.x doesn't respect log path configuration(https://github.com/grafana/grafana/issues/2533). It creates log on default path.